addons/godot-xr-tools/functions/Function_Glide_movement.gd

```diff
@@ -49,27 +49,21 @@ export var horizontal_slew_rate := 1.0
 export var vertical_slew_rate := 2.0
 
 ## Left ARVR Controller
-export (NodePath) var left_controller = null
+export var left_controller := NodePath()
```

Comment:
Should we keep these or should we just rely on get_left_controller and get_right_controller?

Reply:
The new JumpDetect just uses get_left_controller and get_right_controller, so I think these can go. I'll push an update with these removed.

Reply:
Removed the unnecessary NodePath exports.